cmd_test.go

```diff
@@ -147,10 +149,12 @@ func TestHelp(t *testing.T) {
 	t.Parallel()
```

Comment:
Since this test changes the environment variable, which might influence other tests when running in `Parallel()`, I suggest:

1. remove the `t.Parallel()` call.
2. remember the current SHELL env, and defer the restoration of that value.

Reply:
Good catch! Not very thorough from me :(